packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -115,6 +115,10 @@ export function EntitiesSavedStatesExtensible( {
 		'description'
 	);
 
+	const selectItemsToSaveDescription = dirtyEntityRecords.length
```

Comment:
```suggestion
	const selectItemsToSaveDescription = !! dirtyEntityRecords.length
```

Let's use a boolean for the condition.